### ISSUE TITLE
Add instructions yo modal UI for page ID

### ISSUE THREAD

**Issue by DHaussermann** [mattermost/mattermost-plugin-confluence#37]
Ideally the subscribe modal would contain some explanation of what goes into the ID field. 

There is not much space to work with but ideally there could be  text explaining that the ID is numerical and how to find it.

**Comment by jfrerich** [comment]
I understand the sentiment here, but also kind of expect a sysadmin to be able to figure this out without explicitly telling them how.  @aaronrothschild, thoughts?  

![image](https://user-images.githubusercontent.com/7575921/87595528-4dce4c80-c6b4-11ea-95d4-e5cf7e618e03.png)

**Comment by jfrerich** [comment]
I also agree there is not much space available to include extra text.  I'm 2/5 we leave as is.

**Comment by jfrerich** [comment]
@aaronrothschild can you provide your opinion on this issue?  Perhaps change the `Enter the page id.` help text?  As mentioned above, my preference is to leave the modal as is.  Perhaps add additional notes to the README documentation?

**Comment by aaronrothschild** [comment]
Do we have a help text option?  Perhaps show a linkt ot he documentation that goe sinto more detail?
